pkg/edition/java/lite/counter.go

```diff
@@ -0,0 +1,28 @@
+package lite
+
+import (
+	"sync/atomic"
+)
+
+// ConnectionCounter tracks the number of active connections in LiteMode
+var ConnectionCounter = &counter{}
```

Comment:
We do not put global state like this. A process could run many Gate instances at once, they must have their own counter